code-samples/eventing/bookstore-sample-app/db/README.md

```diff
@@ -1,26 +1,109 @@
+```bash
+kubectl apply -f db
+```
+The filenames prefixed with numbers dictate the application order, ensuring Kubernetes orchestrates the resource setup accordingly.
+
+### Step 3: Confirming the Deployment
+Following the application of the deployment files, initialization of the database may require some time. Monitor the deployment's progress by executing:
+```bash
+kubectl get pods -n=default
+```
+A successful deployment is indicated by the `Running` state of the `postgresql-0` pod, as shown below:
+```bash
+NAMESPACE     NAME                     READY   STATUS      RESTARTS   AGE
+default       postgresql-0             1/1     Running     0          1m
+```
+Upon observing the pod in a `Running` state, access the pod using the command:
+```bash
+kubectl exec -it postgresql-0 -n=default -- /bin/bash
+```
+Inside the pod, connect to the database with:
+```bash
+psql -U myuser -d mydatabase
+```
+A successful connection will present you with:
+```bash
+mydatabase=#
+```
+To verify the initialization of the `BookReviews` table, execute:
+```
+mydatabase=# \dt
+```
+If the output lists the `BookReviews` table as follows, your database has been correctly initialized:
+```bash
+ List of relations
+ Schema |     Name     | Type  | Owner  
+--------+--------------+-------+--------
+ public | book_reviews | table | myuser
+(1 row)
+```
+
+## Question & Discussion
+1. Why did we choose to deploy our PostgreSQL database using a StatefulSet instead of a Knative Service?
+
+We use [StatefulSet](https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/) for databases instead of Knative Service mainly because databases need to remember data (like a notebook that keeps your notes). StatefulSets are good at remembering things because they can save data and have a special name and place where they live. This is very important for databases.
+
+Knative Services are more like notebooks that you use and then throw away when you're done. They're great for tasks that don't need to keep data for a long time. You can make them go away when you don't need them and come back when you do. But databases need to always remember information, so they can't just disappear and come back.
+
+Also, databases often talk in their own special language, not the usual web language (HTTP) that Knative Services are really good at understanding. Because of this, Knative Services aren't the best choice for databases. That's why we choose StatefulSet for databases in Kubernetes.
+
+---
+Note box: However, Knative Service supports Volumes and Persistent Volumes, which can be used to store data. You can read more [here](https://knative.dev/docs/serving/services/storage/) about how to use Volumes and Persistent Volumes with Knative Services specially for your use case.
+
+---
+
+2. When should I use Knative Service, and what would be the best use case for it?
+
+You can read more about the best use cases for Knative Service [here](https://knative.dev/docs/serving/samples/)!

```

Comment:
I don't think the samples page describes the point here

Reply:
Yes, those are accurate, however, my point is that we can have a dedicated page for this vs being part of something unrelated, in this page, we can stay concise and link to the dedicated page on "when to use ksvc vs when not to use ksvc"